src/go/rdctl/cmd/shell.go

```diff
@@ -99,3 +110,57 @@ func setupLimaHome() error {
 	os.Setenv("LIMA_HOME", candidatePath)
 	return nil
 }
+
+const restartDirective = "Either run 'rdctl start' or start the Rancher Desktop application first"
+
+func checkLimaIsRunning(commandName string) bool {
+	output, err := exec.Command(commandName, "ls", "0", "--format", "{{.Status}}").CombinedOutput()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Failed to run 'rdctl shell': %s\n", err)
+		return false
+	}
+	if strings.HasPrefix(string(output), "Running") {
+		return true
+	}
+
+	if output == nil || strings.Contains(string(output), "No instance matching 0 found") {
+		fmt.Fprintf(os.Stderr, "The Rancher Desktop VM needs to be created.\n%s.\n", restartDirective)
+	} else {
+		fmt.Fprintf(os.Stderr,
+			"The Rancher Desktop VM needs to be in state \"Running\" in order to execute 'rdctl shell', but it is currently in state \"%s\".\n%s.\n", strings.TrimRight(string(output), "\n"), restartDirective)
+	}
+	return false
+}
+
+func checkWSLIsRunning(distroName string) bool {
+	cmd := exec.Command("wsl", "-l", "-v")
+	output, err := cmd.CombinedOutput()
```

Comment:
Do we need to grab `stderr` too? We don't ever parse anything from there, right?

Reply:
Correct, we don't. If there are errors from running `wsl -l -v` things are pretty bad.